src/pushsource/_impl/backend/errata_source/errata_client.py

```diff
@@ -1,16 +1,45 @@
-import logging
+from functools import wraps, partial
+import gssapi
+import os
+import re
+import requests
+import requests_gssapi
+import subprocess
+import tempfile
 import threading
-from functools import partial
-
+from urllib.parse import urljoin
 import xmlrpc.client as xmlrpc_client
+
+import logging
+
 from more_executors import Executors
 from more_executors.futures import f_zip, f_map
 
 from ...compat_attr import attr
 
+
+
 LOG = logging.getLogger("pushsource.errata_client")
 
 
+USE_HTTP_CLIENT = os.environ.get("PUSHSOURCE_USE_ERRATA_HTTP_API") is not None
```

Comment:
I don't think it should be "is not None" because then even setting the variable to a blank string or to "0" will cause it to be enabled, which is unusual.

It should probably be `== "1"`.